packages/ui-react/src/containers/AnnouncementProvider/AnnoucementProvider.tsx

```diff
@@ -0,0 +1,41 @@
+import { PropsWithChildren, createContext, useCallback, useContext, useState } from 'react';
+
+import FormFeedback from '../../components/FormFeedback/FormFeedback';
+
+export type AriaAnnouncerVariant = 'success' | 'error' | 'warning' | 'info';
+type Announcement = { message: string; variant: AriaAnnouncerVariant };
+type AnnounceFn = (message: string, variant: AriaAnnouncerVariant, duration?: number) => void;
+type ContextValue = { announce: AnnounceFn };
+
+const AriaAnnouncerContext = createContext<ContextValue | undefined>(undefined);
+
+export const useAriaAnnouncer = () => {
+  const announcer = useContext(AriaAnnouncerContext);
+
+  if (!announcer) throw new Error('Announcer context is not defined');
+
+  return announcer.announce;
+};
+
+export const AriaAnnouncerProvider = ({ children }: PropsWithChildren) => {
+  const [announcement, setAnnouncement] = useState<Announcement | null>(null);
+
+  const announce = useCallback((message: string, variant: AriaAnnouncerVariant, duration = 1000) => {
+    setAnnouncement({ message, variant });
+
+    setTimeout(() => {
+      setAnnouncement(null);
+    }, duration);
```

Comment:
The duration is never used in the call. Am I correct? I think it can be quite trivial, because the duration is dependant on the sentence length. Which could also be different in different languages.. I think it's saver to not provide this as an option

Reply:
As we've discussed i've added a longer duration (hardcoded) to give the screenreader some leeway when announcing the states.